mobly/controllers/attenuator.py

```diff
@@ -64,7 +63,7 @@ def create(configs):
     attenuation_device = module.AttenuatorDevice(
         path_count=len(config[KEY_PATHS]))
     attenuation_device.model = attenuator_model
-    instances = attenuation_device.open(config[KEY_ADDRESS], config[KEY_PORT])
```

Comment:
Fix according to Flake8 rule [F841](https://www.flake8rules.com/rules/F841.html)
> F841 local variable 'instances' is assigned to but never used